### PR TITLE
feat(auditlog): Add Trial and Plan Events to Audit Log

### PR DESCRIPTION
**Pull request by ehfeng**
Adding trial and plan event types for audit log entries so hosted Sentry can include these in customer audit logs.

See https://github.com/getsentry/sentry/pull/6617

### CODE REVIEW

**Review by tkaemming — DISMISSED** [review]
Blocking this until GH-6617 is merged.

> If we continue this pattern we should reevaluate, though.

– @tkaemming, Dec 1, 2017, 3:10 PM PST

File: src/sentry/models/auditlogentry.py
Comment (tkaemming) [inline src/sentry/models/auditlogentry.py]: Didn't validate this is actually part of the `data` payload

**Review by tkaemming — APPROVED** [review]
🆒

**Comment by ehfeng** [comment]
Ok, I'm going to try and open the `getsentry` PR that creates these audit log entires before merging this in, just to be safe.

**Comment by tkaemming** [comment]
Sounds like a plan 👌

**Comment by ehfeng** [comment]
Ignoring code coverage because the tests would be within our other repo. Would need to use the manager pattern to fix this.

**Pull request merged**
